hadoop-tools/hadoop-distcp/src/main/java/org/apache/hadoop/tools/DistCpConstants.java

```diff
@@ -122,6 +122,8 @@ private DistCpConstants() {
   /* DistCp CopyListing class override param */
   public static final String CONF_LABEL_COPY_LISTING_CLASS = "distcp.copy.listing.class";
 
+  public static final String CONF_LABEL_DISTCP_SYNC_CLASS = "distcp.sync.class";
```

Comment:
javadocs to explain what it does. the distcp markdown docs will need to mention it too, maybe, or at least add a section about working with ozone saying "read the ozone docs"

Reply:
Removed this approach for separate distcp sync class implementation, instead the correct filesystem implementation will be initialised at run time and getSnapshotDiffReport method would be called.